generateCarouselNav.js

```diff
@@ -1,7 +1,10 @@
-import { ui } from './script.js'
+import {
+	ui,
+	removeActiveImageClass,
+	removeActiveNavButtonClass,
+} from './script.js'
```

Comment:
Nitpick: This file could probably have a bit more meaningful name instead of `script` 😃